contentcuration/contentcuration/views/pwa.py

```diff
@@ -0,0 +1,26 @@
+from django.contrib.staticfiles.storage import staticfiles_storage
+from django.views.generic.base import TemplateView
+
+
+class ServiceWorkerView(TemplateView):
+    content_type = "application/javascript"
+    template_name = "pwa/service_worker.js"
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        # Code to access the compiled serivce worker in developer mode
+        # if getattr(settings, "DEBUG", False):
+        #     import requests
+
+        #     request = requests.get("http://127.0.0.1:4000/dist/serviceWorker.js")
+        #     content = request.content
+        # else:
+        with open(staticfiles_storage.path("static/studio/serviceWorker.js")) as f:
```

Comment:
Should this path contain `static`? I see `static/static/studio/serviceWorker.js` being generated

Reply:
And actually here is the full generated path `studio2/contentcuration/static/static/studio/serviceWorker.js`, it's also missing an additional `contentcuration`?

Reply:
Ah - this is what I'm running into as well - the `serviceWorker.js` file is 404 because of what @bjester mentioned

Reply:
D'oh! Yes... I got a little too static with my static.

Reply:
Ohhh... the reason the extra `contentcuration` is missing is because it is assuming that collectstatic has been run. Will have to use static files finders to make this work with the devserver.